araproc/framework/data_visualization.py

```diff
@@ -117,7 +117,7 @@ def plot_waveform_bundle(
     plt.close(fig)
     del fig, axd
 
-def plot_skymap(the_map = None,
+def plot_skymap(st,list_of_landmarks = None,cal_pulse_index = None,spice_depth = None,the_map = None,
```

Comment:
Can you add documentation for this function?

Reply:
Like the usual description, parameters, returns stuff

Reply:
sure!